### PR TITLE
Streamline create pure materials

### PR DESCRIPTION
**Pull request by gonuke**
This is a first step towards a more streamlined and reviewable script to create pure materials.

It has not been tested!

It should do the following:
* offer two methods: one that creates by mass and one that creates by atom
* handle the (single) special case of a molecular_mass being set
* have a single place to define a Li-6 enrichment and creates a material string using that value

### CODE REVIEW

**Comment by MicahGale** [comment]
Thinking out loud here:

It had been mentioned to go to a JSON format for the data, or at the least a file that just defines all the values. Would it be good testing practice to create a test fixture that reads in that data runs this and then compares the values? It does feel like Ouroboros testing, but I can't think of how to effectively test this without keeping two copies of the data.

**Comment by FusionSandwich** [comment]
In regards to adding molar mass  "[atoms_per_moleculefloat, optional
Number of atoms per molecule of material. Needed to obtain proper scaling of molecular mass. For example, this value for water is 3.0.](https://pyne.io/pyapi/material.html)" Should I add a requirement then that users define atoms to per molecule?

**Comment by Edgar-21** [comment]
To write these materials to OpenMC (and I would guess MCNP?) material formats they all should have a material number assigned to the metadata as well, anyone averse to just numbering these sequentially?

**Comment by Edgar-21** [comment]
Scratch that, I think the write json method seems to add material numbers sequentially by default, conveniently enough

**Comment by gonuke** [comment]
@FusionSandwich - the better way to propose changes to a PR is to make another PR that is based on the original PR branch.

I'm not sure I understand/agree with all the changes you've made in the last commit and by directly committing, we don't have an opportunity for a real review.

File: pureMaterials/createPurematlib.py [hunk 1]
```diff
@@ -554,69 +459,23 @@ def main():
...
     
     # remove lib
     try: 
         os.remove("PureFusionMaterials_libv1.h5")
+        os.remove("PureFusionMaterials_libv1.json")
     except: 
         pass  
         
     # write fnsf1d material library
     mat_lib.write_hdf5("PureFusionMaterials_libv1.h5") # don't set datapath,nucpath...will be pyne default values
```
Comment (Edgar-21) [inline pureMaterials/createPurematlib.py:477]:
```suggestion
```

File: pureMaterials/createPurematlib.py [hunk 2]
```diff
@@ -554,69 +459,23 @@ def main():
...
+    for mat_name, mat_input in mat_data.items():
+        if 'nucvec' in mat_input:
+            mat_lib[mat_name] = make_mat(mat_input['nucvec'], mat_input['density'], mat_input['citation'], 
+                                         mat_input.get('molecular_mass'))
+        if 'atom_frac' in mat_input:
+            mat_lib[mat_name] = make_mat_from_atom(mat_input['atom_frac'], mat_input['density'], mat_input['citation'])
     
     # remove lib
     try: 
         os.remove("PureFusionMaterials_libv1.h5")
```
Comment (Edgar-21) [inline pureMaterials/createPurematlib.py:471]:
```suggestion
```

**Comment by Edgar-21** [comment]
Just a couple loose ends that I should have taken care of in #14, then this lgtm

**Pull request merged**